src/controllers/SendController.php

```diff
@@ -38,7 +38,15 @@ public function actionIndex()
         $submission->fromEmail = $request->getBodyParam('fromEmail');
         $submission->fromName = $request->getBodyParam('fromName');
         $submission->subject = $request->getBodyParam('subject');
-        $submission->message = $request->getBodyParam('message');
+
+        if ($settings->includeAllData) {
+            $submission->message = "<p>Name: {$request->getBodyParam('fromName')}</p>".
```

Comment:
`message` should be set to Markdown-formatted text, not HTML. Each of these `<p>`s should be regular lines of text, separated by two `\n`s. Or perhaps a bullet list would work better here (begin each line with a `-` and separate them with single `\n`s).